model/docs/getting_started.qmd

```diff
@@ -204,6 +204,114 @@ Now, let's investigate the output, particularly the posterior distribution of th
 out = model1.plot_posterior(var="Rt")
 ```
 
+We can use [ArviZ](https://www.arviz.org/) package to create model diagnostics and visualizations. We start by converting the fitted model to ArviZ InferenceData object:
+
+```{python}
+#| label: convert-inference-data
+import arviz as az
+idata = az.from_numpyro(model1.mcmc)
+```
+
+Below we use `plot_trace` to inspect the trace of the first $20$ $R_t$ estimates.
+
+```{python}
+#| label: fig-trace-Rt
+#| fig-cap: Trace plot of Rt posterior distribution
+plt.rcParams['figure.constrained_layout.use'] = True
+
+az.plot_trace(
+    idata.posterior,
+    var_names=['Rt'],
+    coords={'Rt_dim_0': np.arange(19)},
+    compact=False,
+);
+```
+
+and now the remaining $R_t$ estimates:
+
+```{python}
+#| label: fig-trace-Rt2
+#| fig-cap: Trace plot of Rt posterior distribution
+az.plot_trace(
+    idata.posterior,
+    var_names=['Rt'],
+    coords={'Rt_dim_0': np.arange(19,34)},
+    compact=False,
+);
+```
+
+We inspect the posterior distribution of $R_t$ by plotting the $90 \%$ and $50 \%$ highest density interval:
+
+```{python}
+#| label: fig-hdi-Rt
+#| fig-cap: High density interval for Rt estimates
+x_data = idata.posterior['Rt_dim_0']
+y_data = idata.posterior['Rt']
+
+fig, axes = plt.subplots(figsize=(6,5))
+az.plot_hdi(
+    x_data,
+    y_data,
+    hdi_prob=0.9,
+    color='C0',
+    fill_kwargs={'alpha':0.3},
+    ax=axes,
+)
+
+az.plot_hdi(
+    x_data,
+    y_data,
+    hdi_prob=0.5,
+    color='C0',
+    fill_kwargs={'alpha':0.6},
+    ax=axes,
+)
+
+#Add mean of the posterior to the figure
+mean_Rt = az.summary(idata.posterior['Rt'])['mean']
+axes.plot(x_data,mean_Rt, color='C0', label='Mean $R_t$')
```

Comment:
In other plots, you just used `label="Mean"`. For consistency, either always add the label or never add it.

Reply:
changed the label for consistency